common-libs/common/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 2.0.0-beta.5 (2020-08-23)
```

Comment:
@cosminm-affnd should be `2.0.0-beta.6` here ?

Reply:
oups, updated